prism/src/main/java/org/apache/falcon/service/EntitySLAMonitoringService.java

```diff
@@ -303,14 +303,14 @@ public void init() throws FalconException {
 
         freq = StartupProperties.get().getProperty("entity.sla.lookAheadWindow.millis", "900000");
         lookAheadWindowMillis = Integer.parseInt(freq);
-        LOG.debug("No old state exists at: {}, Initializing a clean state.", filePath.toString());
+        LOG.info("Initializing EntitySLAMonitoringService from ", filePath.toString());
```

Comment:
Agree. However, I still see some dependencies on filePath. We should do a clean remove of it from all places including properties file. Lets handle that in a separate JIRA